### PR TITLE
Update Screenshot

### PR DESCRIPTION
**Pull request by ChristinFrohne**
Fixes gitpod-io/website#74

### CODE REVIEW

File: src/docs/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ comprehensive, extensible, and easy to use. With deep GitHub integration, tools
 sharing, and a focus on usability, Gitpod contains your entire dev workflow in a
 browser tab.
 
-![](./images/gitpod-prefix.png)
```
Comment (nisarhassan12) [inline src/docs/index.md:13]: Can you please delete the old screenshot from `./images` as well?

Comment (JesterOrNot) [reply src/docs/index.md:13]: Also, this needs an alt for accessibility and SEO, i.e. for screen-readers

File: src/docs/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ comprehensive, extensible, and easy to use. With deep GitHub integration, tools
...
+![](./images/prefix-lense.png)
```
Comment (JesterOrNot) [inline src/docs/index.md:13]: Add some alt text here. This is to improve SEO and to make our site accessible to people who can't see and need the assistance of a screen-reader.

File: src/docs/index.md (hunk 1 above)
Comment (nisarhassan12) [reply src/docs/index.md:13]: Thanks @JesterOrNot  Great Catch alt is very important for accessibility. I completely forgot about it :slightly_smiling_face:

Comment (ChristinFrohne) [reply src/docs/index.md:13]: True, thanks! Could you point me to where I can change the alt-tag? I'm not so familiar with the flow and couldn't find it.

File: src/docs/index.md (hunk 2 above)
Comment (ChristinFrohne) [reply src/docs/index.md:13]: Thanks for pointing it out. Updating these images is a bit more complex, as my screenshots look different and then the text needs to be adjusted as well. We can close #74 with this PR and leave #510 open to still be fixed.

**Comment by ChristinFrohne** [comment]
Thanks for your review! Updating the other images is a bit more complex as my screenshots look different to Jan's and therefore the text needs to be updated as well. I'd suggest closing #74 with this PR and leave #510  open to adjust the other images.

File: src/docs/index.md (hunk 1 above)
Comment (nisarhassan12) [reply src/docs/index.md:13]: @ChristinFrohne you can add the alt-text inside of the square brackets for instance `Prefix Lense`
```

File: src/docs/index.md (hunk 2 above)
Comment (nisarhassan12) [inline src/docs/index.md:13]:
```suggestion
![Prefix Lense](./images/prefix-lense.png)
```

**Pull request merged**